### ISSUE TITLE
VAPID

### ISSUE THREAD

**Issue by tunnela** [MatthieuLemoine/push-receiver#16]
First I would like to thank you Matthieu for such a great project!

Have you thought about adding support for [VAPID](https://tools.ietf.org/html/rfc8292)?

**Comment by MatthieuLemoine** [comment]
Yeah could be great! If you or someone else wants to tackle this, feel free to contribute.
